viz/metrics-api/prometheus.go

```diff
@@ -46,6 +46,8 @@ const (
 	serverNameLabel        = model.LabelName("srv_name")
 	authorizationKindLabel = model.LabelName("authz_kind")
 	authorizationNameLabel = model.LabelName("authz_name")
+	routeKindLabel         = model.LabelName("route_kind")
+	routeNameLabel         = model.LabelName("route_name")
```

Comment:
nit, tioli: would it make sense for the metric label names to be constants somewhere in this package, so that when theyre referenced elsewhere, such as in `stat_summary.go`, we can know we're using the same string without any typos? not a big deal.